### ISSUE TITLE
Dict deserializer error

### ISSUE THREAD

**Issue by jmolinski** [ramonhagenaars/jsons#41]
Hi!

```
>>> jsons.load({'1': 2}, Dict[str, int])
Traceback (most recent call last):
  File "/home/glen/prywatne/traktpy/venv/lib/python3.7/site-packages/jsons/_main_impl.py", line 132, in load
    return deserializer(json_obj, cls, **kwargs_)
TypeError: 'NoneType' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/glen/prywatne/traktpy/venv/lib/python3.7/site-packages/jsons/_main_impl.py", line 136, in load
    raise DeserializationError(str(err), json_obj, cls)
jsons.exceptions.DeserializationError: 'NoneType' object is not callable
>>> 
```

Self-explanatory :)
v0.8.5

**Comment by ramonhagenaars** [comment]
Thanks jmolinski! This will be fixed soon in ``0.8.6``.

**Comment by ramonhagenaars** [comment]
Version [0.8.6](https://pypi.org/project/jsons/) is now released.
